### ISSUE TITLE
Wiki entry for `contents:`?

### ISSUE THREAD

**Issue by tegob** [kaitai-io/kaitai_struct_formats#4]
What are the various ways of specifying `contents`? I couldn’t find info on `contents` in the wiki.

The example in the README has this:

```ksy
types:
  header:
    seq:
      - id: magic
        contents: 'GIF'
```

Which made me wonder, does it only accept string values or can you specify byte sequences as well? E.g.

```
contents: 0xCA 0xFE
```

Does something like this work?

**Comment by tegob** [comment]
--

**Update:** Ah, [`java_class.ksy`](https://github.com/kaitai-io/kaitai_struct_formats/blob/master/executable/java_class.ksy) answers this question:

```js
contents: [0xca, 0xfe, 0xba, 0xbe]
```

**Comment by GreyCat** [comment]
Yeah, definitely a missing point! Thanks, I'll try to fix it ASAP.

**Comment by GreyCat** [comment]
Here you are: https://github.com/kaitai-io/kaitai_struct/wiki/Attribute-description#contents
Is it good / clear enough?
